nav2_recovery_plugins/package.xml

```diff
@@ -12,7 +12,7 @@
   <build_depend>nav2_common</build_depend>
 
   <build_depend>rclcpp</build_depend>
-  <build_depend>nav2_recoveries</build_depend>
+  <build_depend>nav2_recovery</build_depend>
```

Comment:
Should nav2_recovery be a dependency of the plugins? I don't think so.

Reply:
Is this because the Recovery.hpp is still in the nav2_recovery package?

Reply:
The plugins implement the ```recovery.hpp``` interface from ```nav2_recovery```, should be a dependency right?

Reply:
There's an abstract interface ```recovery.hpp``` defined in ```nav2_core``` which is implemented by the recovery interface in ```nav2_recovery```, I'm not sure I understand why.

Reply:
the recovery nav2_core is just the interface, the recovery in nav2_recovery implements an action server and a bunch of other stuff to make new recoveries easy to make. 

OK, we can keep this for now.